src/vanilla/ClientModelExtensions.cs

```diff
@@ -346,7 +347,8 @@ public static string CheckNull(string valueReference, string executionBlock)
             if (sequence != null && sequence.ShouldValidateChain())
             {
                 var elementVar = scope.GetUniqueName("element");
-                var innerValidation = sequence.ElementType.ValidateType(scope, elementVar, null);
+                variable.Name = elementVar;

```

Comment:
Why are you assigning the name here?

Reply:
Previously we were directly passing `elementVar` to `ValidateType` method. Now that we are not passing `string` that's why I had to set `elementVar` to `variable.Name`.